microsoft/testsuites/lis/lissuite.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy import assert_that
+
+from lisa import Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.tools import Wget
+from lisa.util import SkippedException
+
+
+@TestSuiteMetadata(
+    area="lis",
+    category="functional",
+    description="""
+    ##################################
+    """,
+)
+class lis(TestSuite):  # noqa
+    @TestCaseMetadata(
+        description="""
+            #################
+        """,
+        priority=1,
+    )
+    def verify_lis_driver_version(self, node: Node) -> None:
+        wget_tool = node.tools[Wget]
+        rpm_status = node.execute(
```

Comment:
Add operating system as a requirement.

Reply:
I put the requirement on the test suite instead of the test case. Is that ok?

Reply:
It's ok, if the requirement is the same to all test cases. If some test cases are different, it can be rewritten at test case level also.